tests/test_atlas.py

```diff
@@ -11,58 +11,66 @@ def atlas():
 
 
 def test_initialization(atlas):
-    assert atlas.metadata == {"name": "test",
-                              "citation": "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007",
-                              "atlas_link": "www.brain-map.org.com",
-                              "species": "Mus musculus",
-                              "symmetric": True,
-                              "resolution": [100, 100, 100],
-                              "shape": [132, 80, 114]}
-
-
-@pytest.mark.parametrize("attribute, val", [("shape", [132, 80, 114]),
-                                            ("resolution", [100, 100, 100]),
-                                            ("name", "test"),
-                                            ("symmetric", True)])
+    assert atlas.metadata == {
+        "name": "test",
+        "citation": "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007",
+        "atlas_link": "www.brain-map.org.com",
+        "symmetric": True,
```

Comment:
Yup, this is why I would like to have the atlas generation scripts within the same repo, so that we can test those consistencies together. I will move them back in my next PR